### PR TITLE
Add CNCF Community Group to assets.md

### PR DESCRIPTION
**Pull request by danielgblanco**
Adds [OpenTelemetry CNCF Community Group](https://community.cncf.io/opentelemetry/) to assets. Please note that the CNCF Community account for cncf-opentelemetry-governance@lists.cncf.io has not yet been created (or added as Lead Organizer).

### CODE REVIEW

File: assets.md [hunk 1]
```diff
@@ -90,6 +90,20 @@ Link: https://pypi.org/user/opentelemetry/
 
 ## Communication channels
 
+### CNCF Community Group
+
+Used by the End-User SIG to organize events (e.g. OTel Q&A, OTel in Practice, feedback sessions, etc.).
+
+CNCF are the owners of the group, so certain requests (e.g. adding organizers) need to be done via Service Desk.
+
+Link: https://community.cncf.io/opentelemetry/
```
Comment (danielgblanco) [inline assets.md:99]: Pending discussion on this group possibly being renamed to _OpenTelemetry End-Users_.

**Review by arminru — APPROVED** [review]

**Review by svrnm — APPROVED** [review]

**Review by trask — APPROVED** [review]

**Review by austinlparker — APPROVED** [review]

File: assets.md (hunk 1 above)
Comment (jpkrohling) [reply assets.md:102]: If the End-User SIG is OK in having other SIGs use it for their purposes in the future as well, I'm fine with having it as "OpenTelemetry".

**Review by jpkrohling — APPROVED** [review]

File: assets.md (hunk 1 above)
Comment (danielgblanco) [reply assets.md:102]: That's a good point. I assume these events would be public (as in, aimed at the general community) events regardless, so hopefully benefit from input/help from the End-User SIG to publicise them?

Comment (danielgblanco) [reply assets.md:102]: After today's GC meeting, this conversation can be resolved and group name remains "OpenTelemetry"

**Comment by danielgblanco** [comment]
Please note that after today's GC meeting we have decided that the GC does not need to have an account with admin@opentelemetry.io and can request changes, if needed, via Service Desk.

**Pull request merged**